src/services/business-lookup-services.ts

```diff
@@ -57,6 +57,10 @@ export default class BusinessLookupServices {
       // filter out results without a valid identifier
       return results.filter(result => {
         const pattern = /^[A-Z]{1,3}\d{7}$/
+        // Check if status is HISTORICAL and result.status is active, exclude it
+        if (!status && result.status === 'ACTIVE') {
+          return false
+        }
```

Comment:
This isn't ideal since the search returns a default number of entries and this might drop them all.

Try using the `status` parameter (as per line 36) where this is called from.

Reply:
Hi @severinbeauvais  @jamespaologarcia 

I checked the dev and local.
The dev site doesn't include the `state` query param when sending the request to the API
But local includes the `state` query on `feature-way-of-navigating` br. 

I am not sure why they are diff


<img width="1423" alt="Screenshot 2023-10-04 at 1 53 28 PM" src="https://github.com/bcgov/namerequest/assets/54437031/a5d802b2-7053-4b64-8aa2-46604e538417">
<img width="1433" alt="Screenshot 2023-10-04 at 1 55 53 PM" src="https://github.com/bcgov/namerequest/assets/54437031/5d596d81-7530-4563-80a6-037c1b6b4e0a">

Reply:
Are you searching for a business to create the same type of NR (eg, restoration)?

Reply:
Weird.

Reply:
Yes, I did same NR type

Reply:
Might have to do with current OCP issues related to authentication.

I do see here that the BusinessLookup _should_ be passed pass HISTORICAL or ACTIVE:
https://github.com/bcgov/namerequest/blob/07dff9c9ee9fee00ba291506523497957fc9921f/src/components/new-request/search-components/business-lookup-fetch.vue#L99
then
https://github.com/bcgov/namerequest/blob/07dff9c9ee9fee00ba291506523497957fc9921f/src/components/new-request/search-components/business-lookup-fetch.vue#L10
then
https://github.com/bcgov/namerequest/blob/07dff9c9ee9fee00ba291506523497957fc9921f/src/components/new-request/business-lookup.vue#L120

Reply:
James, you can explain it to the other reviewers.

I assume you're going to revert this code so I'm resolving this conversation.